research/picai/data_utils.py

```diff
@@ -169,28 +178,29 @@ def get_img_and_seg_paths(
             torch tensor that give the class proportions.
     """
 
-    # load datasheets
-    file_name = f"PI-CAI_train-fold-{fold_id}.json" if train else f"PI-CAI_val-fold-{fold_id}.json"
+    # Make sure at least one sequence will be present
+    assert not (include_t2w is False and include_adc is False and include_hbv is False)
+
+    # load dataset overview
+    file_name = f"train-fold-{fold_id}.json" if train else f"val-fold-{fold_id}.json"
     file_path = os.path.join(overviews_dir, file_name)
     with open(Path(file_path)) as fp:
         file_json = json.load(fp)
 
+    # Determine valid file name endings (indicator for sequence type: T1, T2, ADC)
+    all_postfix = ("0000.nii.gz", "0001.nii.gz", "0002.nii.gz")
+    valid_postfix = tuple([item for flag, item in zip([include_t2w, include_adc, include_hbv], all_postfix) if flag])
+
     # load paths to images and labels
-    img_paths = [[os.path.join(base_dir, path) for path in path_list] for path_list in file_json["image_paths"]]
-    seg_paths = [os.path.join(base_dir, path) for path in file_json["label_paths"]]
+    img_paths = [
+        [path for path in path_list if path.endswith(valid_postfix)] for path_list in file_json["image_paths"]
+    ]
+    seg_paths = [path for path in file_json["label_paths"]]
 
     # Determine class proportions
     class_ratio = [int(np.sum(file_json["case_label"])), int(len(img_paths) - np.sum(file_json["case_label"]))]
     class_proportions = class_ratio / np.sum(class_ratio)
 
-    # Log dataset information
```

Comment:
Is this no longer helpful? You could use the flower logging instead of prints if that is the issue?

Reply:
Yeah, good call! On second thought it would definitely be beneficial to log this information